<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LAPTOP-T6OJV50P" name="online_tusur.unit_online_tusur.NewTestNG1" tests="6" failures="0" timestamp="2022-02-04T18:57:13 MSK" time="0.018" errors="1">
  <testcase name="testEmptyCollection" time="0.002" classname="online_tusur.unit_online_tusur.NewTestNG1"/>
  <system-out/>
  <testcase name="testAge1" time="0.010" classname="online_tusur.unit_online_tusur.NewTestNG1"/>
  <system-out/>
  <testcase name="testAge2" time="0.002" classname="online_tusur.unit_online_tusur.NewTestNG1"/>
  <system-out/>
  <testcase name="testExceptCollection" time="0.001" classname="online_tusur.unit_online_tusur.NewTestNG1"/>
  <system-out/>
  <testcase name="testItemCollection" time="0.003" classname="online_tusur.unit_online_tusur.NewTestNG1">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at online_tusur.unit_online_tusur.NewTestNG1.testItemCollection(NewTestNG1.java:45)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testItemCollection -->
  <system-out/>
  <testcase name="testNotEmptyCollection" time="0.000" classname="online_tusur.unit_online_tusur.NewTestNG1">
    <skipped/>
  </testcase> <!-- testNotEmptyCollection -->
  <system-out/>
</testsuite> <!-- online_tusur.unit_online_tusur.NewTestNG1 -->
